packages/webview_flutter/lib/webview_flutter_tizen.dart

```diff
@@ -792,18 +822,14 @@ mixin _PlatformViewGestureMixin on RenderBox implements MouseTrackerAnnotation {
   set hitTestBehavior(PlatformViewHitTestBehavior value) {
     if (value != _hitTestBehavior) {
       _hitTestBehavior = value;
-      if (owner != null) {
-        markNeedsPaint();
-      }
+      if (owner != null) markNeedsPaint();
```

Comment:
```suggestion
      if (owner != null) { 
        markNeedsPaint();
      }
```

Reply:
Well ... This is also from original code.

Reply:
@bwikbs Are you sure? :P

https://github.com/flutter/flutter/blob/02c026b03cd31dd3f867e5faeb7e104cce174c5f/packages/flutter/lib/src/rendering/platform_view.dart#L696-L698

Reply:
Oh, that curly brace is not of the if statement. My mistake.

Reply:
It's a slightly different story, but the change in the number of line in a sentence is the result of  ``vscode``. 
I'm wondering what settings of ``vscode`` will not change original part.